easybuild/easyconfigs/c/cma/cma-2.7.0-intel-2019a.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PythonPackage'
+
+name = 'cma'
```

Comment:
not `pycma`?

I see that using `cma` instead of `pycma` makes the easyconfig simpler, because of the `PYPI` url and module name, so just checking

Reply:
if it was up to me I would go for `pycma` because I prefer lowercase names and `cma` is a TLA ;)

Reply:
Based on the README in https://github.com/CMA-ES/pycma, `pycma` makes sense:

```
pycma is a Python implementation of CMA-ES and a few related numerical optimization tools.
```